### ISSUE TITLE
Language zh-tw disappeared

### ISSUE THREAD

**Issue by NeKoOuO** [ImranR98/Obtainium#1737]
There's no Traditional Chinese in language option

**Comment by ImranR98** [comment]
Yes, it was contributed but could not be added due to this issue: https://github.com/aissat/easy_localization/issues/372
